### PR TITLE
Bump version to 5.2.2

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,7 +53,7 @@ gem "logstash-filter-fingerprint"
 gem "logstash-filter-geoip"
 gem "logstash-filter-grok"
 gem "logstash-filter-json"
-gem "logstash-filter-kv"
+gem "logstash-filter-kv", "< 4.0.0"
```
Comment (suyograo) [inline Gemfile]: KV 4.0.0 is not BWC; changes a config option

**Comment by andrewvc** [comment]
LGTM

**Comment by suyograo** [comment]
@andrewvc approved on Slack

**Comment by jordansissel** [comment]
Thsi should have also been merged into 5.3 and 5.x

**Pull request merged**